flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/table/MySqlTableSourceFactory.java

```diff
@@ -101,6 +102,7 @@ public DynamicTableSource createDynamicTableSource(Context context) {
         Duration connectTimeout = config.get(CONNECT_TIMEOUT);
         int connectMaxRetries = config.get(CONNECT_MAX_RETRIES);
         int connectionPoolSize = config.get(CONNECTION_POOL_SIZE);
+        long outputRateLimit = config.get(OUTPUT_RATE_LIMIT);
```

Comment:
Maybe we can invork `OUTPUT_RATE_LIMIT` validate method here like `DATABASE_NAME` and `startupOptions` config.